02-introducing-survey-data.Rmd

```diff
@@ -1,2 +1,188 @@
+
+
+### Weighting {#overview-post-weighting}
+
+Weighting can typically be used to address some of the error sources identified in the previous sections.  For example, weights may be used to address coverage, sampling, and nonresponse errors and many published surveys will include an "analysis weight" variable that combines these adjustments. However, weighting itself can also introduce *adjustment error*, so researchers need to balance which types of errors should be corrected with weighting. The construction of weights is outside the scope of this book and researchers should reference other materials if interested constructing their own (e.g., {CITE Jill's Book}). Instead, this book assumes the survey has been completed, weights are constructed, and data is made available for users. We will walk users through how to read documentation and work with the data and analysis weights provided to analyze and interpret survey results correctly.
+
+#### Example: Number of Pets in a Household {- #overview-post-weighting-ex}
+
+In the simple example of our survey, we decided to use a stratified sample by state to select our sample members.  Knowing this sampling design our researcher can include selection weights for analysis that account for how the sample members were selected into the survey.  Additionally, the sampling frame may have the type of building associated with each address, so we could include the building type as a potential nonresponse weighting variable, along with some interviewer observations that may be related to our research topic of average number of pets in a household.  Combining these weights we could create an analytic weight that researchers can use when analyzing the data.
+
+
+### Disclosure {#overview-post-disclosure}
+
+Before data is allowed to be made publicly available, researchers will need to ensure that individual respondents can not be identified by the data.  There are a variety of different methods that can be used, but one common method is swapping records.  Researchers may swap specific data values across different respondents such that it does not impact insights that come from the data, but ensures that specific individuals cannot be identified.  Other methods may include aggregating various categories of responses or location information to avoid having only a few people in a given group that could be identified.  There is as much art as there is science to the methods used for disclosure, and in documentation researchers should only provide a high level comments that disclosure was conducted and not specific details to ensure nobody can reverse the disclosure and thus identify individuals.  For more information on different disclosure methods please see {CITE??--Stephanie?}.
```

Comment:
```suggestion
Before data is allowed to be made publicly available, researchers will need to ensure that individual respondents can not be identified by the data when confidentiality is required.  There are a variety of different methods that can be used, including data swapping, top or bottom coding, coarsening, and perturbation.  In data swapping, researchers may swap specific data values across different respondents such that it does not impact insights that come from the data, but ensures that specific individuals cannot be identified.  For extreme values, top and bottom coding is sometimes used. For example, very high earners may have their incomes listed as $99,999,999 for any household reporting $99,999,999 or more for their income which is an example of top-coding. Other methods may include aggregating various categories of responses or location information to avoid having only a few people in a given group that could be identified such as including only categorical income instead of continuous which is a form of coarsening.  Data producers may also perturb the data by adding random noise. There is as much art as there is science to the methods used for disclosure, and in documentation researchers should only provide a high level comments that disclosure was conducted and not specific details to ensure nobody can reverse the disclosure and thus identify individuals.  For more information on different disclosure methods please see {CITE??--Stephanie?}.
```